### ISSUE TITLE
How to tune parameters to make converge faster

### ISSUE THREAD

**Issue by nliu86** [Microsoft/CNTK#2145]
The training converged very fast at the beginning, but eventually ground to a halt, moving only very little in the end. Is there any way to tune parameters to speed up training in the end? 

Here is the training log:
<img width="1330" alt="trainlog" src="https://user-images.githubusercontent.com/13305896/28450804-808bc642-6d9e-11e7-9095-f8a72182e6ce.png">

Full log:
[trainlog.txt](https://github.com/Microsoft/CNTK/files/1164587/trainlog.txt)

And config:
[cntk.training.parallel.txt](https://github.com/Microsoft/CNTK/files/1164574/cntk.training.parallel.txt)

**Comment by cha-zhang** [comment]
You should consider reducing the learning rate when training stagnates. You can use a learning rate schedule for that.

**Comment by nliu86** [comment]
@cha-zhang Thanks! Aren't moments and rmsprop already taking care of adjusting learning rates automatically? Is this normal behavior that training slows down when it's close to global optimal?

**Comment by cha-zhang** [comment]
Yes, it's a normal behavior. If you look at the CNTK examples, most of them use learning rate schedules that reduces over epochs. 

In the future, please ask similar questions on Stack Overflow since it's about CNTK usage rather than a CNTK issue.
